CONTRIBUTING.md

```diff
@@ -51,9 +51,42 @@ Once we've discussed your changes and you've got your code ready, make sure that
 
 ## Contribution Guidelines for Documentation
 
+* pre-commit
+
+    A framework for managing and maintaining multi-language pre-commit hooks.
+    Pre-commit can be [installed](https://pre-commit.com/#installation) with `pip`, `curl`, `brew` or `conda`.
```

Comment:
I still believe it's better to add a universal script for contributors to easily use it, like using `curl` to do the installation.

Reply:
Most people would not install `pre-commit` with `curl` if they had the choice to use `pip`, `brew` or `conda`.  I use Homebrew and I update my packages everyday and it makes it so much easier to use and manage.